iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NodeBreakerVoltageLevel.java

```diff
@@ -647,6 +653,21 @@ public int getNode2() {
                     });
         }
 
+        @Override
+        public void removeInternalConnections(int node1, int node2) {
+            int[] internalConnectionsToBeRemoved = Arrays.stream(graph.getEdges())
+                    .filter(e -> graph.getEdgeObject(e) == null)
+                    .filter(e -> graph.getEdgeVertex1(e) == node1 && graph.getEdgeVertex2(e) == node2)
```

Comment:
I think you should also get edges between node2 and node1 (the graph is not directed)